build.xml

```diff
@@ -147,7 +147,7 @@
       <bootclasspath refid="pearplanner.module.bootclasspath"/>
       <classpath refid="pearplanner.module.production.classpath"/>
       <classpath>
-        <path refid="external.classpath" />
+      	<path refid="external.classpath" />
```

Comment:
Whitespace only changes normally belong in their own commit.  This one should be reverted because it also makes the indentation inconsistent.